testing/integration/src/builtin.rs

```diff
@@ -0,0 +1,103 @@
+use std::collections::BTreeMap;
+
+use anyhow::{Context, Result};
+use cid::Cid;
+use futures::executor::block_on;
+use fvm::state_tree::{ActorState, StateTree};
+use fvm::{init_actor, system_actor};
+use fvm_ipld_blockstore::{Blockstore, MemoryBlockstore};
+use fvm_ipld_car::load_car;
+use fvm_ipld_encoding::CborStore;
+use fvm_shared::actor::builtin::{load_manifest, Type};
+use fvm_shared::version::NetworkVersion;
+use multihash::Code;
+
+use crate::error::Error::{
+    FailedToLoadManifest, FailedToSetActor, FailedToSetState, MultipleRootCid, NoCidInManifest,
+};
+
+const BUNDLES: [(NetworkVersion, &[u8]); 2] = [
+    (NetworkVersion::V14, actors_v6::BUNDLE_CAR),
+    (NetworkVersion::V15, actors_v7::BUNDLE_CAR),
+];
```

Comment:
This is fine for now, but I think test will want to provide the bundles they care about in the long term.